### ISSUE TITLE
V2905: Broken links (File not found)

### ISSUE THREAD

**Issue by v-maudel** [aspnet/EntityFramework.Docs#1201]
Hello, @divega ,  
Translator has reported possible source content issue. 
**Link to Github file:** https://github.com/aspnet/EntityFramework.Docs/blob/live/entity-framework/ef6/modeling/code-first/data-types/enums.md
**Row where the issue is located:**  24
**Comment:** Broken links for demonstration videos

Can you please review the comment and provide answer/explanation? Also, if source fix is needed please provide link to the PR that will include the fix, so we can confirm and close this Issue.
Many thanks in advance.

**Comment by divega** [comment]
I was able to chase down the missing  videos. It turns out that the downloads were unpublished some time ago. I have requested that the ownership of the downloads is transferred to me. 

The following are unpublished, so the links are broken:

bda37c87-1b28-45e1-8df9-1e74b5facdee - Entity Framework Enum Support – Code First
72405c2f-3451-4411-b086-98c798a7c72d - Entity Framework Spatial Support – Code First
3898aa92-5096-47aa-887b-6c74ef0c0605 - Entity Framework Enum Support - EF Designer
90a6481f-c267-4c55-ae81-a92f01e881b8 - Entity Framework Spatial - EF Designer
db0c6d07-e33f-4a6b-a6dd-3c2787f135b0 - Entity Framework Table Valued Functions (TVFs)
48b9c7eb-44f1-4091-9405-33a27e096eb7 - Entity Framework Multiple Diagrams per Model

The following are published but I have also requested ownership transfer:

631b5bb5-d64f-4b5d-a78f-2c8c82c23a60 - Entity Framework Development Workflows
96c09ead-3156-451c-b216-3dc84fd87bd8 - Entity Framework Database First
1dd0c821-1568-469e-83cb-f3e2e86e4057 - Entity Framework Model First
3d7fba9e-25e8-4624-acb9-4995b72e7fbc - Entity Framework Code First to a New_Database

This one also exists and seems to be an unpublished duplicate of the previous one. It can probably be deleted: 

d3cc8067-9888-4957-b69b-6c51a20160ef - Entity Framework Code First to a New Database
